exla/mix.exs

```diff
@@ -57,8 +57,8 @@ defmodule EXLA.MixProject do
 
   defp deps do
     [
-      # {:nx, path: "../nx"},
-      {:nx, "~> 0.2.0"},
+      {:nx, path: "../nx"},
```

Comment:
This is needed for FFT to be implemented in both EXLA and Torchx (in their respective mix.exs files)

Reply:
Note I have already added a path depednency on main. :)